codalab/lib/bundle_cli.py

```diff
@@ -815,8 +814,8 @@ def collapse_bare_command(argv):
         """
         try:
             i = argv.index('---')
-            # Convert the command after '---' to a shell-escaped version of the string.
-            shell_escaped_command = [quote(x) for x in argv[i + 1 :]]
+            # Convert empty string after '---' to a shell-escaped version of the string.
+            shell_escaped_command = [shlex.quote(x) if not x else x for x in argv[i + 1 :]]
```

Comment:
Why are we not quoting anymore?  If I do `--- echo "Hello world"`, "Hello world" still needs to be quoted.

Reply:
That's true